### PR TITLE
SW-1690 Add import and export labels to buttons

### PR DESCRIPTION
**Pull request by alexTerraformation**
Screen Shot 2022-10-25 at 11 44 46 AM
Screen Shot 2022-10-25 at 11 44 54 AM
Screen Shot 2022-10-25 at 11 45 15 AM
Screen Shot 2022-10-25 at 11 46 17 AM
Screen Shot 2022-10-25 at 11 47 55 AM

### CODE REVIEW

File: src/components/emptyStatePages/EmptyStateContent.tsx [hunk 1]
```diff
@@ -47,7 +47,7 @@ const useStyles = makeStyles((theme: Theme) => ({
   }),
   listItem: {
     flex: '1 1 auto',
-    maxWidth: '200px',
+    maxWidth: '205px',
```
Comment (alexTerraformation) [inline src/components/emptyStatePages/EmptyStateContent.tsx:50]: With the icon the "Import Inventory" string was just slightly too long to fit in a button with padding and this maxWidth.  Open to suggestions for a different way to handle this.

File: src/components/emptyStatePages/EmptyStateContent.tsx [hunk 2]
```diff
@@ -135,7 +135,7 @@ export default function EmptyStateContent(props: EmptyStateContentProps): JSX.El
       <div className={classes.listContainer}>
         {listItems.map((item) => {
           return (
-            <Grid item xs={gridSize()} key={item.title} className={`${classes.listItem}`}>
+            <Grid item xs={12} key={item.title} className={`${classes.listItem}`}>
```
Comment (alexTerraformation) [inline src/components/emptyStatePages/EmptyStateContent.tsx:138]: Oops, will fix this :)

File: src/components/emptyStatePages/EmptyStateContent.tsx (hunk 1 above)
Comment (constanzauanini) [reply src/components/emptyStatePages/EmptyStateContent.tsx:50]: This looks good to me

File: license-report.html [hunk 3]
```diff
@@ -45,6 +45,6 @@
...
...</td><td class="string">perpetual</td><td class="string">material</td><td class="string">MIT</td><td class="string">git+https://github.com/timothygu/utm.git</td><td class="string">1.1.1</td><td class="string">1.1.1</td><td class="string">^1.1.1</td><td class="string">Timothy Gu <timothygu99@gmail.com></td></tr><tr><td class="string">terraware</td><td class="string">terraware</td><td class="string">web-vitals</td><td class="string">perpetual</td><td class="string">material</td><td class="string">Apache-2.0</td><td class="string">git+https://github.com/GoogleChrome/web-vitals.git</td><td class="string">1.1.2</td><td class="string">1.1.2</td><td class="string">^1.0.1</td><td class="string">Philip Walton philip@philipwalton.com http://philipwalton.com</td></tr><tr><td class="string">terraware</td><td class="string">terraware</td><td class="string">worker-loader</td><td class="string">perpetual</td><td class="string">material</td><td class="string">MIT</td><td class="string">git+https://github.com/webpack-contrib/worker-loader.git</td><td class="string">3.0.8</td><td class="string">3.0.8</td><td class="string">^3.0.8</td><td class="string">Tobias Koppers @sokra</td></tr></tbody></table>
```
[the remark this answers is not in the document]
Comment (alexTerraformation) [reply license-report.html:48]: Yeah I ran license-report.  Do we not usually do that when updating a package?

[the remark this answers is not in the document]
Comment (constanzauanini) [reply license-report.html:48]: I don't, but makes sense to run it

**Review by constanzauanini — APPROVED** [review]

**Pull request merged**